pkg/services/accesscontrol/resourcepermissions/store.go

```diff
@@ -17,8 +18,8 @@ import (
 	"github.com/grafana/grafana/pkg/util"
 )
 
-func NewStore(sql db.DB, features featuremgmt.FeatureToggles, actionsetService *ActionSetService) *store {
-	return &store{sql, features, *actionsetService}
+func NewStore(sql db.DB, features featuremgmt.FeatureToggles, actionsetService ActionSetService) *store {
```

Comment:
what is the reason for not having a pointer? 🤔

Reply:
Mostly to make the test setup easier (we can just pass in nil if the action set service is not being tested). But I can undo this change if you think it's going to cause issues.

Reply:
I realised that we don't even need to pass actin set service into the resource permission store, so I've refactored it further. Let me know what you think :)